Sources/System/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ target_sources(SystemPackage PRIVATE
   Internals/Mocking.swift
   Internals/Syscalls.swift
   Internals/WindowsSyscallAdapters.swift)
-target_link_libraries(SystemPackage PRIVATE
+target_link_libraries(SystemPackage PUBLIC
```

Comment:
Just for my own education, what does this do?

Reply:
This means that any user of `SystemPackage` will now transitively link against `CSystem`, which in this case propagates the include directives for the module definition (as it is an interface library).  Because `SystemPackage` publicly imports `CSystem` and forwards certain items as `@_emitIntoClient`, we need to ensure that all clients of `SystemPackage` can see `CSystem`.